ec2-metadata

```diff
@@ -23,6 +23,7 @@ Options:
 -l/--ami-launch-index     The index of this instance in the reservation (per AMI).
 -m/--ami-manifest-path    The manifest path of the AMI with which the instance was launched.
 -n/--ancestor-ami-ids     The AMI IDs of any instances that were rebundled to create this AMI.
+-D/--aws-domain           The root domain name that AWS uses in this region
```

Comment:
From a naming convention perspective, what made you choose to prefix with `aws-`?